openmetadata-ui/src/main/resources/ui/src/components/TeamDetails/TeamDetailsV1.tsx

```diff
@@ -874,6 +874,7 @@ const TeamDetailsV1 = ({
                     Add Role
                   </ButtonAntd>
                   <ListEntities
+                    disableRemoveBtn={!entityPermissions.EditAll}
```

Comment:
Can we not use negative conditions here?
Directly pass the value and check it in the child component.

Reply:
Made the suggested changes